### DISCUSSION TITLE
An exception occurred while executing a query: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'definitionModificationDate' in 'field list'

### DISCUSSION THREAD

**Discussion by baladpi** [pimcore/pimcore#17376] [General]
I am facing this issue while upgrade to 11.2.1, is there anyway to resolve this or is it recommended to upgrade to 2024.2 version. How to do upgrade to the latest version

**Comment by agdrs** [comment]
Hi @baladpi 
we were facing this issue too and used the commands pimcore:build:classes and pimcore:deployment:classes-rebuild --create-classes and that worked for us.

**Reply by baladpi** [reply]
For me it worked with bin/console doctrine:migrations:migrate

**Reply by devvmaaster** [reply]
you are my lifesaver :-)

**Reply by lucastrahlendorff** [reply]
Can confirm. Had this issue randomly appearing after a changed class definition but the doctrine migration solved it. Thx!
